### ISSUE TITLE
They all become an invisible tab stop

### ISSUE THREAD

**Issue by mrmsbarnes** [MicrosoftDocs/windows-uwp#828]
The phrase "Make a set of related controls a single tab stop" is a bit jarring.  Surely the word "use" or "tab order" or "accessible by" should be in there somewhere.  Make a control group accessible by a single tab stop.  Still not quite right, but I really don't want my controls transforming into something they are not.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7538142a-74f2-63fe-5ffc-b205a69f9493
* Version Independent ID: c382db31-6f36-b071-423a-72b61b77aed1
* Content: [Keyboard interactions - UWP app developer](https://docs.microsoft.com/en-us/windows/uwp/design/input/keyboard-interactions)
* Content Source: [windows-apps-src/design/input/keyboard-interactions.md](https://github.com/MicrosoftDocs/windows-uwp/blob/docs/windows-apps-src/design/input/keyboard-interactions.md)
* Product: **windows**
* GitHub Login: @Karl-Bridge-Microsoft
* Microsoft Alias: **kbridge**

**Comment by Karl-Bridge-Microsoft** [comment]
thanks for the suggestions @mrmsbarnes. I've updated the topic.
